plugins/woocommerce-blocks/package.json

```diff
@@ -359,12 +359,13 @@
 					"name": "JavaScript",
 					"command": "test:js",
 					"changes": [
-						"jest.config.js",
```

Comment:
Blocks doesn't have a root jest.config.js. the config is in the tests/js folder. (I am Readding those files in https://github.com/woocommerce/woocommerce/pull/47105/commits/37180f01737430f5267d3ef91722970fe620ab55 )